op-challenger/README.md

```diff
@@ -154,7 +155,7 @@ If the game is resolved successfully, the result is printed.
 ```shell
 ./bin/op-challenger list-games \
   --l1-eth-rpc <L1_ETH_RPC> \
-  --network <Predefined_Network>
+  --game-factory-address <GAME_FACTORY_ADDRESS>
```

Comment:
We want to use `--network` here as it's much more straight-forward. The devnet has to specify the factory address directly because it's an ephemeral chain and so isn't in the superchain registry.

Reply:
Ah makes sense. Was following the pattern for list-games, but I get now that the game_factory is a single hard-coded address per network, so easier to use indeed, whereas its not possible for list-games.

Will change the description of the options below then, which we describing the GAME_FACTORY_ADDRESS instead of NETWORK